### PR TITLE
Fixes GetResourceCultureName for embedded resources

### PR DESCRIPTION
**Pull request by kirthik**
cc @davidfowl

### CODE REVIEW

File: src/Microsoft.Dnx.Compilation.CSharp.Common/project.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "compilationOptions": { "define": [ "TRACE" ], "allowUnsafe": true, "warningsAsErrors": true },
     "dependencies": {
         "Microsoft.CodeAnalysis.CSharp": "1.1.0-beta1-*",
+        "Microsoft.Dnx.Compilation": "1.0.0-*",
```
Comment (davidfowl) [inline src/Microsoft.Dnx.Compilation.CSharp.Common/project.json]: Uh nope

Comment (kirthik) [inline src/Microsoft.Dnx.Compilation.CSharp.Common/project.json]: Ok. I will remove it and add the code directly.

**Comment by davidfowl** [comment]
When the ci passes :shipit:

**Comment by kirthik** [comment]
fixed 5bbb84bea40d8835da0c3eefd5a2343b50347084

**Pull request closed without merging**